src/test/java/stepDefinition/ConfirmSteps.java

```diff
@@ -0,0 +1,30 @@
+package stepDefinition;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.Then;
+import org.junit.Assert;
+import pageObjects.login.AppointmentConfirmPage;
+import utility.GetPropertiesValue;
+
+import java.util.List;
+import java.util.Map;
+
+public class ConfirmSteps {
+    private GetPropertiesValue getPropertiesValue = new GetPropertiesValue();
+    @Then("^user sees Appointment Confirmation page as below$")
+    public void userSeesAppointmentConfirmationPageAsBelow(DataTable table) {
+        List<Map<String, String>> data = table.asMaps(String.class, String.class);
+        for(int i=0;i<data.size();i++){
+            Assert.assertEquals(AppointmentConfirmPage.facilitys().get(i).getText(),data.get(i).get("facility"));
+            Assert.assertEquals(AppointmentConfirmPage.actions().get(i).getText(),data.get(i).get("action"));
+            Assert.assertEquals(AppointmentConfirmPage.HealthcarePrograms().get(i).getText(),data.get(i).get("HealthcareProgram"));
+            Assert.assertEquals(AppointmentConfirmPage.dates().get(i).getText(),data.get(i).get("date"));
+            Assert.assertEquals(AppointmentConfirmPage.comments().get(i).getText(),data.get(i).get("comment"));
+        }
```

Comment:
vì table chỉ có 1 giá trị nên không cần phải dùng for đâu, có thể viết như này luôn: 
```
 Assert.assertEquals(AppointmentConfirmPage.facilitys().getText(),data.get(0).get("facility"));
Assert.assertEquals(AppointmentConfirmPage.actions().getText(),data.get(0).get("action"));   Assert.assertEquals(AppointmentConfirmPage.HealthcarePrograms().getText(),data.get(0).get("HealthcareProgram"));
Assert.assertEquals(AppointmentConfirmPage.dates().getText(),data.get(0).get("date"));
Assert.assertEquals(AppointmentConfirmPage.comments().getText(),data.get(0).get("comment"));
```